docs/user/alerting/alerting-troubleshooting.asciidoc

```diff
@@ -199,3 +199,19 @@ In addition to the above methods, broadly used the next approaches and common is
 include::troubleshooting/alerting-common-issues.asciidoc[]
 include::troubleshooting/event-log-index.asciidoc[]
 include::troubleshooting/testing-connectors.asciidoc[]
+
+
+[float]
+[[alerting-kibana-disable]]
+=== Temporarily disable
+
+A cluster may become unresponsive or sluggish if too many or expensive {kib} rules 
+are attempting to run. As a stop gap measure, you may consider temporarily overriding
+the {kib} Task Manager to gain breathing room to resolve your situation by restarting 
```

Comment:
is "breathing room" plain enough?  Seems kinda american-ish, but not sure.  Maybe something more descriptive would be better anyway?  Something like 

> As a stop gap measure, you may consider temporarily overriding the {kib} Task Manager to have it run fewer tasks less frequently.  This should provide some time to make changes to the system to resolve your situation.  The relevant {kib} configuration keys are:

Reply:
Thanks for putting this together Stef!
I agree, we should definitely avoid figures of speech like that.
Perhaps @gchaps can help us find a good way of expressing this?

Reply:
I can take a look.  IMO we can make this more concise with something like this:
> To temporarily reduce the workload, you can change the Task Manager settings to run fewer tasks less frequently: